src/daemon/auth/common/permissions.rs

```diff
@@ -67,6 +67,9 @@ iterable_enum! {
         ROUTES_READ,
         ROUTES_UPDATE,
         ROUTES_ANALYSIS,
+        ASPAS_READ,
```

Comment:
It's been a while since I looked at the code but I assume we need to add these new permissions to `defaults/roles.polar`.

Reply:
added